apis/kubedb/v1alpha2/pgbouncer_types.go

```diff
@@ -71,9 +71,9 @@ type PgBouncerSpec struct {
 	// +optional
 	PodTemplate ofst.PodTemplateSpec `json:"podTemplate,omitempty"`
 
-	// Databases to proxy by connection pooling.
+	// Database to proxy by connection pooling.
 	// +optional
-	Databases []Databases `json:"databases,omitempty"`
+	Database Database `json:"database,omitempty"`
```

Comment:
Is it really optional ?

Reply:
If it is compulsory, remove the `+optional` tag.
if it is optional, make it pointer type `Database *Database`

Reply:
+optional tag removed